region/brinstar/red.json

```diff
@@ -1645,17 +1645,32 @@
           "to": [
             {
               "id": 3,
-              "requires": [
-                {"obstacle":{
-                  "id": "A",
-                  "requires": [
-                    {"or":[
-                      {"enemyDamage": {
-                        "enemy": "Sidehopper",
-                        "type": "contact",
-                        "hits": 4
-                      }},
-                      {"and":[
+              "strats": [
+                {
+                  "name": "Power Beam Sidehopper Kill",
```

Comment:
Pretty much yeah. It's in there because it's doable, and because maybe knowing it's not a softlock could have some kind of value someday. Though the energy requirement makes that a big maybe.

There's certainly no logical reason to enter this room without being able to get the item.